src/xtgeo/cube/_cube_import.py

```diff
@@ -116,22 +115,24 @@ def import_segy(self, sfile):
         logger.debug("XTGeo rotation is %s", rotation)
 
     # attributes to update
-    self._ilines = ilines
-    self._xlines = xlines
-    self._ncol = ncol
-    self._nrow = nrow
-    self._nlay = nlay
-    self._xori = xori
-    self._xinc = xinc
-    self._yori = yori
-    self._yinc = yinc
-    self._zori = zori
-    self._zinc = zinc
-    self._rotation = rotation
-    self.values = values
-    self._yflip = yflip
-    self._segyfile = sfile
-    self._traceidcodes = traceidcodes
+    return {
+        "ilines": ilines,
+        "xlines": xlines,
+        "ncol": ncol,
+        "nrow": nrow,
+        "nlay": nlay,
+        "xori": xori,
+        "xinc": xinc,
+        "yori": yori,
+        "yinc": yinc,
+        "zori": zori,
+        "zinc": zinc,
+        "rotation": rotation,
+        "values": values,
+        "yflip": yflip,
+        "segyfile": sfile,
```

Comment:
I would consider dropping this key to decouple the dict from the fact that it was read form a file. It made sense (I guess) to store it as part of a Cube-object, but when returning a standard dict-object it does not IMO. The caller knows this filename and can store it explicitly in the dict if required.

Reply:
I would too, however the segyfile setter/getter combo is part of the public api now, so removing it would mean a breaking change.